### PR TITLE
[ROX-10236] Link deployments and namespace table

### PR DESCRIPTION
**Pull request by md2119**
## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- ~Evaluated and added CHANGELOG entry if required~
- ~Determined and documented upgrade steps~
- ~Documented user facing changes~

If any of these don't apply, please comment below.

### CODE REVIEW

File: tools/generate-helpers/pg-table-bindings/store_test.go.tpl [hunk 1]
```diff
@@ -73,6 +73,10 @@ func (s *{{$namePrefix}}StoreSuite) TestStore() {
 	s.False(exists)
 	s.Nil(found{{.TrimmedType|upperCamelCase}})
 
+    {{- /* Check for referenced schema */ -}}
```
Comment (viswajithiii) [inline tools/generate-helpers/pg-table-bindings/store_test.go.tpl:76]: Why?

File: tools/generate-helpers/pg-table-bindings/main.go [hunk 2]
```diff
@@ -213,12 +213,21 @@ func generateSchemaRecursive(schema *walker.Schema, refs []*walker.Schema, visit
...
 	// No top-level schema has a parent unless attached synthetically.
 	for _, parent := range schema.ReferencedSchema {
+		// Code generation may happen in isolation. Therefore, in the current schema, we generate referenced schema.
+		// If the referenced schema is already generated, then prefer the gen of that proto over
+		// other protos generating it.
+		absFilePath := getSchemaFileName(pkgPath, parent.Table)
+		_, err := os.Stat(absFilePath)
+		if err == nil || os.IsExist(err) {
+			continue
+		}
```
Comment (viswajithiii) [inline tools/generate-helpers/pg-table-bindings/main.go:230]: This is okay for now, but I've fixed this issue in my branch.

File: tools/generate-helpers/pg-table-bindings/store_test.go.tpl (hunk 1 above)
Comment (md2119) [reply tools/generate-helpers/pg-table-bindings/store_test.go.tpl:76]: We cannot unit test upserts for tables referencing other tables given our internal directory structure. For such cases, we first need to upsert the referenced object for respect referential constraint. Furthermore, our datastore structure is utterly variables and hence this cannot be autogenerated.

**Comment by md2119** [comment]
hmm, bad rebase I guess

File: tools/generate-helpers/pg-table-bindings/store_test.go.tpl (hunk 1 above)
Comment (viswajithiii) [reply tools/generate-helpers/pg-table-bindings/store_test.go.tpl:81]: Okay I see what you're saying, I ran into this on my PR as well.

**Pull request closed without merging**